packages/client/hmi-client/src/page/project/tera-project.vue

```diff
@@ -104,7 +114,7 @@ const props = defineProps<{
 	project: IProject;
 	assetName?: string;
 	assetId?: string;
-	assetType?: ProjectAssetTypes;
+	assetType?: ProjectAssetTypes | 'overview' | '';
```

Comment:
Maybe create an `IAssetType` in the `/types` folder

Reply:
I think this would be worth dealing with while we merge `ResourceType` and `AssetType`.